### PR TITLE
OCC-128: Documentation menu

### LINKED ISSUE #257: Documentation menu (OCC-128)
You added some new pages to the documentation recently.

Ex: https://commerce.orchardcore.net/en/latest/features/inventory/

if you want them to appear in the menu, you need to add them to the `nav` section in the mkdocs.yml file.

[Jira issue](https://lombiq.atlassian.net/browse/OCC-128)

### PR DESCRIPTION
**Pull request by DAud-IcI**
OCC-128

Fixes #257

### CODE REVIEW

**Review by agriffard — APPROVED** [review]
LGTM.
Will you keep your links in the Readme pointing to GitHub or will you redirect to the docs site?

**Comment by agriffard** [comment]
I temporarily enabled the branch on RTD so you see the result:
https://commerce.orchardcore.net/en/issue-occ-128/

File: mkdocs.yml [hunk 1]
```diff
@@ -74,7 +74,15 @@ plugins:
...
+    - Stripe Payment: features/stripe-payment.md
+    - Taxation: features/taxation.md
+    - User Features: features/user-features.md
   - Resources:
-      - Libraries: resources/libraries/README.md
+    - Libraries: resources/libraries/README.md
   - Releases:
-      - 0.0.1: releases/0.0.1.md
+    - 0.0.1: releases/0.0.1.md
+    - 1.0.0: releases/1.0.0.md
```
[the remark this answers is not in the document]
Comment (DAud-IcI) [reply mkdocs.yml:87]: Okay, fixed.

**Comment by DAud-IcI** [comment]
>  Will you keep your links in the Readme pointing to GitHub or will you redirect to the docs site?

I prefer to keep them pointing to to GitHub, because this way they are relative links that work in different branches and in past revisions just as well.

**Pull request merged**